packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -0,0 +1,321 @@
+              const reachability = spark.meetings.getReachability();
+              assert.notExists(reachability, 'reachability is undefined because #setReachability has not been called');
+            });
+          });
+          describe('after #setReachability', () => {
+            beforeEach(() => {
+              spark.meetings.setReachability();
+              const reachabilityMocker = spark.meetings.getReachability();
+              sinon.stub(reachabilityMocker, 'gatherReachability').returns(true);
+            });
+            it('gets the reachability data instance from spark.meetings', () => {
+              const reachability = spark.meetings.getReachability();
+              assert.exists(reachability, 'reachability is defined because #setReachability has been called');
+              assert.typeOf(reachability, 'object', 'should be an object');
+              assert.instanceOf(reachability, Reachability, 'should be a reachability instance');
+            });
+          });
+        });
+        describe('#getPersonalMeetingRoom', () => {
+          it('gets the personal meeting room instance from spark.meetings', () => {
+            const personalMeetingRoom = spark.meetings.getPersonalMeetingRoom();
+            assert.exists(personalMeetingRoom, 'personal meeting room instance is set up at object creation');
+            assert.typeOf(personalMeetingRoom, 'object', 'should be an object');
+            assert.instanceOf(personalMeetingRoom, PersonalMeetingRoom, 'should be a personal meeting room instance');
+          });
+        });
+        describe('Static shortcut proxy methods', () => {
+          let staticMeetings;
+          beforeEach(() => {
+            staticMeetings = Meetings;
+          });
+          afterEach(() => {
+            staticMeetings = null;
+          });
+          describe('MeetingCollection getByKey proxies', () => {
+            beforeEach(() => {
+              MeetingCollection.getByKey = sinon.stub().returns(true);
+            });
+            describe('#getMeetingByType', () => {
+              it('gets the Meeting instance from MeetingCollection using type and value', () => {
+                staticMeetings.getMeetingByType('test', 'test');
+                assert.calledOnce(MeetingCollection.getByKey);
+                assert.calledWith(MeetingCollection.getByKey, 'test', 'test');
+              });
+            });
+          });
+          describe('MeetingCollection getAll proxies', () => {
+            beforeEach(() => {
+              MeetingCollection.getAll = sinon.stub().returns(true);
+            });
+            describe('#getAllMeetings', () => {
+              it('calls MeetingCollection to get all meetings with supplied options', () => {
+                staticMeetings.getAllMeetings({test: 'test'});
+                assert.calledOnce(MeetingCollection.getAll);
+                assert.calledWith(MeetingCollection.getAll, {test: 'test'});
```

Comment:
Same as above. Insert multiple meetings in before each and assert they are all returned